vllm/transformers_utils/config.py

```diff
@@ -693,7 +693,7 @@ def recurse_elems(elem: Any):
                 "quant_method": "fp8",
                 "activation_scheme": "static"
             }
-        elif quantization.get("quant_method") == "compressed-tensors":
+        elif quantization.get("quant_method") == "compressed_tensors":
```

Comment:
```suggestion
        elif quantization.get("quant_method").replace("-", "_") == "compressed_tensors":
```
Maybe this needs to be replaced in the assignment line too? Not sure the order of operations at this point
Please test with https://huggingface.co/nm-testing/Mistral-Small-3.1-24B-Instruct-2503-FP8 in the mistral format

Reply:
I've opted to go with `compressed-tensors` as the standard instead of `compressed_tensors` so this can stay unchanged